### DISCUSSION TITLE
Welcome to plantuml Discussions!

### DISCUSSION THREAD

**Discussion by hallvard** [hallvard/plantuml#138] [Announcements]
## Welcome!
  We’re introducing Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas, particularly the direction of the PlantUML plugin.
  * Engage with other community members.

Consider discussing ideas or suggestions before creating an issues, to ensure issues are clear, concise and implementable.

**Comment by toddamalvoso** [comment]
Does the eclipse plugin use the online server to render the images?

**Reply by hallvard** [reply]
No, it uses the plantuml library wrapped in an eclipse plugin, which in turn (usually) uses a locally installed graphviz.

**Reply by toddamalvoso** [reply]
Thanks. Just to understand your use of the word "usually", what happens when the locally installed graphviz is not used?

**Reply by hallvard** [reply]
Plantuml now has experimental support for ELK (see the plantuml site) which is a java layout engine. If you don't explicitly turn on ELK, graphviz is used and will need to be installed.
